### PR TITLE
Drop currently unused tags from osm2pgsql style file

### PR DESCRIPTION
**Pull request by math1985**

### CODE REVIEW

**Comment by math1985** [comment]
Note that this is a PR against the hstore branch.

File: openstreetmap-carto.style [hunk 1]
```diff
@@ -21,14 +20,10 @@ node       capital      text         linear
...
 node,way   covered      text         linear
 node,way   culvert      text         linear
-node,way   cutting      text         linear
 node,way   denomination text         linear
-node,way   disused      text         linear
 node       ele          text         linear
-node,way   embankment   text         linear
 node,way   foot         text         linear
 node,way   generator:source    text  linear
-node,way   harbour      text         polygon
```
Comment (pnorman) [inline openstreetmap-carto.style]: this should be changed to phstore, not dropped

Comment (math1985) [inline openstreetmap-carto.style]: Done.

Comment (pnorman) [inline openstreetmap-carto.style]: The same is true for any other tag that indicates an area and will not have a column

Comment (math1985) [inline openstreetmap-carto.style]: Done.

**Pull request merged**